starter/script.js

```diff
@@ -1 +1,233 @@
 // put code here!
+window.alert("Welcome to the Magic Kingdom! The Happiest Place on Earth!" +
```

Comment:
calling `alert` and `prompt` on `window` is unnecessary as `window` is the default context